packages/vscode-extension/src/commands/ListObjects.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from 'vscode';
+import SuiteCloudRunner from '../core/SuiteCloudRunner';
+import CommandsMetadataSingleton from '../service/CommandsMetadataSingleton';
+import { getRootProjectFolder, unwrapExceptionMessage } from '../util/ExtensionUtil';
+import { scloudOutput } from '../extension';
+import { MessageService } from '../service/MessageService';
+
+const objectTypes: [] = require('@oracle/netsuite-suitecloud-nodejs-cli/src/metadata/ObjectTypesMetadata');
```

Comment:
`const objectTypes: { name: string; value: { name: string; type: string; prefix: string; hasRelatedFiles: boolean; relatedFiles?: { type: string}[] } } [] = ...` and you'll have auto-completion and validation when using objectTypes ;)

Reply:
aplied.